2025-07-15 19:40:35.432544+02:00 jdbc[6]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:35.471604+02:00 jdbc[6]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:35.698993+02:00 jdbc[14]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:35.705105+02:00 jdbc[14]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:35.818926+02:00 jdbc[22]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:35.834983+02:00 jdbc[22]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:36.000937+02:00 jdbc[30]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:36.006616+02:00 jdbc[30]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:36.354536+02:00 jdbc[38]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:36.365666+02:00 jdbc[38]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:36.454284+02:00 jdbc[46]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:36.464302+02:00 jdbc[46]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:36.564621+02:00 jdbc[54]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:36.575685+02:00 jdbc[54]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:36.642329+02:00 jdbc[62]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:36.652794+02:00 jdbc[62]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:36.742561+02:00 jdbc[70]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:36.758423+02:00 jdbc[70]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:36.912175+02:00 jdbc[78]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:36.922208+02:00 jdbc[78]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:37.010012+02:00 jdbc[86]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:37.024815+02:00 jdbc[86]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:37.091397+02:00 jdbc[95]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:37.097001+02:00 jdbc[95]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:37.197341+02:00 jdbc[103]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:37.202325+02:00 jdbc[103]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:37.362704+02:00 jdbc[112]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:37.373201+02:00 jdbc[112]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:37.426373+02:00 jdbc[121]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:37.433375+02:00 jdbc[121]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:37.483367+02:00 jdbc[130]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:37.488949+02:00 jdbc[130]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:37.591123+02:00 jdbc[138]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:37.599187+02:00 jdbc[138]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:37.654486+02:00 jdbc[146]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:37.660043+02:00 jdbc[146]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:37.717204+02:00 jdbc[154]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:37.722187+02:00 jdbc[154]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:37.774940+02:00 jdbc[163]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:37.780957+02:00 jdbc[163]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:37.864997+02:00 jdbc[172]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:37.871014+02:00 jdbc[172]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:37.986918+02:00 jdbc[181]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:37.990904+02:00 jdbc[181]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:38.196997+02:00 jdbc[191]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:38.205345+02:00 jdbc[191]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:38.322134+02:00 jdbc[199]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:38.333382+02:00 jdbc[199]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:38.403164+02:00 jdbc[207]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:40:38.410754+02:00 jdbc[207]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:21.928461+02:00 jdbc[6]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:21.955804+02:00 jdbc[6]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:22.197516+02:00 jdbc[14]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:22.216822+02:00 jdbc[14]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:22.402684+02:00 jdbc[22]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:22.417839+02:00 jdbc[22]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:22.654019+02:00 jdbc[30]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:22.669293+02:00 jdbc[30]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:22.929555+02:00 jdbc[38]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:22.939574+02:00 jdbc[38]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:23.064010+02:00 jdbc[46]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:23.071453+02:00 jdbc[46]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:23.196179+02:00 jdbc[54]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:23.208549+02:00 jdbc[54]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:23.316946+02:00 jdbc[62]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:23.329057+02:00 jdbc[62]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:23.423103+02:00 jdbc[70]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:23.449554+02:00 jdbc[70]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:23.636041+02:00 jdbc[78]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:23.640550+02:00 jdbc[78]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:23.701423+02:00 jdbc[86]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:23.723068+02:00 jdbc[86]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:23.801313+02:00 jdbc[95]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:23.823000+02:00 jdbc[95]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:23.927871+02:00 jdbc[103]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:23.938290+02:00 jdbc[103]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:24.008962+02:00 jdbc[112]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:24.031550+02:00 jdbc[112]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:24.151096+02:00 jdbc[121]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:24.165372+02:00 jdbc[121]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:24.287258+02:00 jdbc[130]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:24.295747+02:00 jdbc[130]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:24.401868+02:00 jdbc[138]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:24.444134+02:00 jdbc[138]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:24.572209+02:00 jdbc[146]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:24.588438+02:00 jdbc[146]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:24.679751+02:00 jdbc[154]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:24.705373+02:00 jdbc[154]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:24.833765+02:00 jdbc[163]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:24.956841+02:00 jdbc[163]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:25.066669+02:00 jdbc[172]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:25.107565+02:00 jdbc[172]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:25.232962+02:00 jdbc[181]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:25.252393+02:00 jdbc[181]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:25.405101+02:00 jdbc[191]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:25.418434+02:00 jdbc[191]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:25.474363+02:00 jdbc[199]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:25.477927+02:00 jdbc[199]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:25.602045+02:00 jdbc[207]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:45:25.608074+02:00 jdbc[207]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:40.223691+02:00 jdbc[6]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:40.248166+02:00 jdbc[6]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:40.419113+02:00 jdbc[14]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:40.423607+02:00 jdbc[14]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:40.506361+02:00 jdbc[22]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:40.516098+02:00 jdbc[22]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:40.671298+02:00 jdbc[30]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:40.687341+02:00 jdbc[30]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:40.877545+02:00 jdbc[38]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:40.895256+02:00 jdbc[38]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:41.016079+02:00 jdbc[46]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:41.025443+02:00 jdbc[46]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:41.080778+02:00 jdbc[54]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:41.091875+02:00 jdbc[54]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:41.232767+02:00 jdbc[62]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:41.241932+02:00 jdbc[62]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:41.318733+02:00 jdbc[70]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:41.321727+02:00 jdbc[70]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:41.353169+02:00 jdbc[78]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:41.355028+02:00 jdbc[78]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:41.439359+02:00 jdbc[86]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:41.456352+02:00 jdbc[86]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:41.571873+02:00 jdbc[95]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:41.576920+02:00 jdbc[95]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:41.700922+02:00 jdbc[103]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:41.721028+02:00 jdbc[103]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:41.816223+02:00 jdbc[112]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:41.848257+02:00 jdbc[112]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:41.916267+02:00 jdbc[121]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:41.921988+02:00 jdbc[121]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:41.987580+02:00 jdbc[130]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:41.989571+02:00 jdbc[130]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:42.033047+02:00 jdbc[138]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:42.051710+02:00 jdbc[138]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:42.160349+02:00 jdbc[146]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:42.167481+02:00 jdbc[146]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:42.294726+02:00 jdbc[154]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:42.309408+02:00 jdbc[154]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:42.503347+02:00 jdbc[163]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:42.648910+02:00 jdbc[163]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:42.747436+02:00 jdbc[172]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:42.768598+02:00 jdbc[172]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:42.834580+02:00 jdbc[181]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:42.837940+02:00 jdbc[181]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:42.931190+02:00 jdbc[191]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:42.949400+02:00 jdbc[191]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:43.029752+02:00 jdbc[199]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:43.034684+02:00 jdbc[199]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:43.096694+02:00 jdbc[207]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:46:43.101678+02:00 jdbc[207]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:47:42.723621+02:00 jdbc[6]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-15 19:47:42.739696+02:00 jdbc[6]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-07-31 23:26:46.214991+02:00 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: java.lang.AssertionError: 57344 != 53248 [2.2.224/3]" [50000-224]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1546)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.FileStore.storeBuffer(FileStore.java:1529)
	at org.h2.mvstore.FileStore.lambda$serializeAndStore$3(FileStore.java:1432)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:545)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:328)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1095)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:619)
	at java.base/java.lang.Thread.run(Thread.java:1447)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: java.lang.AssertionError: 57344 != 53248 [2.2.224/3]" [50000-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 12 more
Caused by: org.h2.mvstore.MVStoreException: java.lang.AssertionError: 57344 != 53248 [2.2.224/3]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	... 7 more
Caused by: java.lang.AssertionError: 57344 != 53248
	at org.h2.mvstore.RandomAccessStore.shrinkStoreIfPossible(RandomAccessStore.java:679)
	at org.h2.mvstore.RandomAccessStore.writeChunk(RandomAccessStore.java:377)
	at org.h2.mvstore.RandomAccessStore.writeChunk(RandomAccessStore.java:28)
	at org.h2.mvstore.FileStore.storeBuffer(FileStore.java:1524)
	... 6 more
2025-07-31 23:26:46.215998+02:00 jdbc[31]: exception
org.h2.jdbc.JdbcSQLNonTransientException: General error: "General error: ""org.h2.mvstore.MVStoreException: java.lang.AssertionError: 57344 != 53248 [2.2.224/3]"" [50000-224]" [50000-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.engine.Database.throwLastBackgroundException(Database.java:1817)
	at org.h2.engine.SessionLocal.beforeCommitOrRollback(SessionLocal.java:764)
	at org.h2.engine.SessionLocal.commit(SessionLocal.java:679)
	at org.h2.command.Command.stop(Command.java:158)
	at org.h2.command.CommandContainer.stop(CommandContainer.java:260)
	at org.h2.command.Command.executeUpdate(Command.java:297)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:216)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:174)
	at org.example.dao.TrenoDAO.aggiungiTreno(TrenoDAO.java:21)
	at org.example.persistence.PopolaDB.popola(PopolaDB.java:47)
	at org.example.persistence.PopolaDB.pulisciDatabase(PopolaDB.java:270)
	at FedeltaTest.setUp(FedeltaTest.java:24)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:688)
	at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
	at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:149)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptLifecycleMethod(TimeoutExtension.java:126)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptBeforeEachMethod(TimeoutExtension.java:76)
	at org.junit.jupiter.engine.execution.ExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(ExecutableInvoker.java:115)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.lambda$invoke$0(ExecutableInvoker.java:105)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:104)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:98)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.invokeMethodInExtensionContext(ClassBasedTestDescriptor.java:490)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$synthesizeBeforeEachMethodAdapter$19(ClassBasedTestDescriptor.java:475)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeBeforeEachMethods$2(TestMethodTestDescriptor.java:167)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeBeforeMethodsOrCallbacksUntilExceptionOccurs$5(TestMethodTestDescriptor.java:195)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeBeforeMethodsOrCallbacksUntilExceptionOccurs(TestMethodTestDescriptor.java:195)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeBeforeEachMethods(TestMethodTestDescriptor.java:164)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:127)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:65)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:129)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:127)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:126)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:84)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:143)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:129)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:127)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:126)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:84)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:143)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:129)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:127)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:126)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:84)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:32)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:51)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:108)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:88)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:67)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:52)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:96)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:75)
	at org.apache.maven.surefire.junitplatform.LazyLauncher.execute(LazyLauncher.java:56)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.execute(JUnitPlatformProvider.java:184)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:148)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:122)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
Caused by: org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: java.lang.AssertionError: 57344 != 53248 [2.2.224/3]" [50000-224]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1546)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.FileStore.storeBuffer(FileStore.java:1529)
	at org.h2.mvstore.FileStore.lambda$serializeAndStore$3(FileStore.java:1432)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:545)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:328)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1095)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:619)
	at java.base/java.lang.Thread.run(Thread.java:1447)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: java.lang.AssertionError: 57344 != 53248 [2.2.224/3]" [50000-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 12 more
Caused by: org.h2.mvstore.MVStoreException: java.lang.AssertionError: 57344 != 53248 [2.2.224/3]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	... 7 more
Caused by: java.lang.AssertionError: 57344 != 53248
	at org.h2.mvstore.RandomAccessStore.shrinkStoreIfPossible(RandomAccessStore.java:679)
	at org.h2.mvstore.RandomAccessStore.writeChunk(RandomAccessStore.java:377)
	at org.h2.mvstore.RandomAccessStore.writeChunk(RandomAccessStore.java:28)
	at org.h2.mvstore.FileStore.storeBuffer(FileStore.java:1524)
	... 6 more
2025-07-31 23:26:46.228390+02:00 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: java.lang.AssertionError: 57344 != 53248 [2.2.224/3]" [50000-224]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1546)
	at org.h2.mvstore.FileStore.writeInBackground(FileStore.java:1842)
	at org.h2.mvstore.FileStore$BackgroundWriterThread.run(FileStore.java:2248)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: java.lang.AssertionError: 57344 != 53248 [2.2.224/3]" [50000-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 6 more
Caused by: org.h2.mvstore.MVStoreException: java.lang.AssertionError: 57344 != 53248 [2.2.224/3]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.FileStore.storeBuffer(FileStore.java:1529)
	at org.h2.mvstore.FileStore.lambda$serializeAndStore$3(FileStore.java:1432)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:545)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:328)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1095)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:619)
	at java.base/java.lang.Thread.run(Thread.java:1447)
Caused by: java.lang.AssertionError: 57344 != 53248
	at org.h2.mvstore.RandomAccessStore.shrinkStoreIfPossible(RandomAccessStore.java:679)
	at org.h2.mvstore.RandomAccessStore.writeChunk(RandomAccessStore.java:377)
	at org.h2.mvstore.RandomAccessStore.writeChunk(RandomAccessStore.java:28)
	at org.h2.mvstore.FileStore.storeBuffer(FileStore.java:1524)
	... 6 more
2025-07-31 23:26:46.235930+02:00 jdbc[31]: exception
org.h2.jdbc.JdbcSQLNonTransientException: General error: "General error: ""org.h2.mvstore.MVStoreException: java.lang.AssertionError: 57344 != 53248 [2.2.224/3]"" [50000-224]"; SQL statement:
ROLLBACK [50000-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.engine.Database.throwLastBackgroundException(Database.java:1817)
	at org.h2.engine.SessionLocal.beforeCommitOrRollback(SessionLocal.java:764)
	at org.h2.engine.SessionLocal.rollback(SessionLocal.java:798)
	at org.h2.command.dml.TransactionCommand.update(TransactionCommand.java:49)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:256)
	at org.h2.jdbc.JdbcConnection.rollbackInternal(JdbcConnection.java:1441)
	at org.h2.jdbc.JdbcConnection.close(JdbcConnection.java:380)
	at org.example.dao.TrenoDAO.aggiungiTreno(TrenoDAO.java:13)
	at org.example.persistence.PopolaDB.popola(PopolaDB.java:47)
	at org.example.persistence.PopolaDB.pulisciDatabase(PopolaDB.java:270)
	at FedeltaTest.setUp(FedeltaTest.java:24)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:688)
	at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
	at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:149)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptLifecycleMethod(TimeoutExtension.java:126)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptBeforeEachMethod(TimeoutExtension.java:76)
	at org.junit.jupiter.engine.execution.ExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(ExecutableInvoker.java:115)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.lambda$invoke$0(ExecutableInvoker.java:105)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:104)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:98)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.invokeMethodInExtensionContext(ClassBasedTestDescriptor.java:490)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$synthesizeBeforeEachMethodAdapter$19(ClassBasedTestDescriptor.java:475)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeBeforeEachMethods$2(TestMethodTestDescriptor.java:167)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeBeforeMethodsOrCallbacksUntilExceptionOccurs$5(TestMethodTestDescriptor.java:195)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeBeforeMethodsOrCallbacksUntilExceptionOccurs(TestMethodTestDescriptor.java:195)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeBeforeEachMethods(TestMethodTestDescriptor.java:164)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:127)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:65)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:129)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:127)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:126)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:84)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:143)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:129)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:127)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:126)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:84)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:143)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:129)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:127)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:126)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:84)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:32)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:51)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:108)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:88)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:67)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:52)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:96)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:75)
	at org.apache.maven.surefire.junitplatform.LazyLauncher.execute(LazyLauncher.java:56)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.execute(JUnitPlatformProvider.java:184)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:148)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:122)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
Caused by: org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: java.lang.AssertionError: 57344 != 53248 [2.2.224/3]" [50000-224]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1546)
	at org.h2.mvstore.FileStore.writeInBackground(FileStore.java:1842)
	at org.h2.mvstore.FileStore$BackgroundWriterThread.run(FileStore.java:2248)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: java.lang.AssertionError: 57344 != 53248 [2.2.224/3]" [50000-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 6 more
Caused by: org.h2.mvstore.MVStoreException: java.lang.AssertionError: 57344 != 53248 [2.2.224/3]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.FileStore.storeBuffer(FileStore.java:1529)
	at org.h2.mvstore.FileStore.lambda$serializeAndStore$3(FileStore.java:1432)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:545)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:328)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1095)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:619)
	at java.base/java.lang.Thread.run(Thread.java:1447)
Caused by: java.lang.AssertionError: 57344 != 53248
	at org.h2.mvstore.RandomAccessStore.shrinkStoreIfPossible(RandomAccessStore.java:679)
	at org.h2.mvstore.RandomAccessStore.writeChunk(RandomAccessStore.java:377)
	at org.h2.mvstore.RandomAccessStore.writeChunk(RandomAccessStore.java:28)
	at org.h2.mvstore.FileStore.storeBuffer(FileStore.java:1524)
	... 6 more
2025-07-31 23:26:46.270623+02:00 jdbc[32]: 
org.h2.message.DbException: The database has been closed [90098-224]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.engine.SessionLocal.getTransaction(SessionLocal.java:1610)
	at org.h2.engine.SessionLocal.getStatementSavepoint(SessionLocal.java:1620)
	at org.h2.engine.SessionLocal.setSavepoint(SessionLocal.java:858)
	at org.h2.command.Command.executeUpdate(Command.java:248)
	at org.h2.engine.Engine.openSession(Engine.java:259)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:343)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:125)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:613)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:199)
	at org.example.persistence.DBConnectionSingleton.getConnection(DBConnectionSingleton.java:20)
	at org.example.dao.TrenoDAO.aggiungiTreno(TrenoDAO.java:13)
	at org.example.persistence.PopolaDB.popola(PopolaDB.java:55)
	at org.example.persistence.PopolaDB.pulisciDatabase(PopolaDB.java:270)
	at FedeltaTest.setUp(FedeltaTest.java:24)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:688)
	at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
	at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:149)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptLifecycleMethod(TimeoutExtension.java:126)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptBeforeEachMethod(TimeoutExtension.java:76)
	at org.junit.jupiter.engine.execution.ExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(ExecutableInvoker.java:115)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.lambda$invoke$0(ExecutableInvoker.java:105)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:104)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:98)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.invokeMethodInExtensionContext(ClassBasedTestDescriptor.java:490)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$synthesizeBeforeEachMethodAdapter$19(ClassBasedTestDescriptor.java:475)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeBeforeEachMethods$2(TestMethodTestDescriptor.java:167)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeBeforeMethodsOrCallbacksUntilExceptionOccurs$5(TestMethodTestDescriptor.java:195)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeBeforeMethodsOrCallbacksUntilExceptionOccurs(TestMethodTestDescriptor.java:195)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeBeforeEachMethods(TestMethodTestDescriptor.java:164)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:127)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:65)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:129)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:127)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:126)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:84)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:143)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:129)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:127)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:126)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:84)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:143)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:129)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:127)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:126)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:84)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:32)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:51)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:108)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:88)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:67)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:52)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:96)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:75)
	at org.apache.maven.surefire.junitplatform.LazyLauncher.execute(LazyLauncher.java:56)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.execute(JUnitPlatformProvider.java:184)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:148)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:122)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: The database has been closed [90098-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:690)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 88 more
Caused by: org.h2.mvstore.MVStoreException: java.lang.AssertionError: 57344 != 53248 [2.2.224/3]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.FileStore.storeBuffer(FileStore.java:1529)
	at org.h2.mvstore.FileStore.lambda$serializeAndStore$3(FileStore.java:1432)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:545)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:328)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1095)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:619)
	at java.base/java.lang.Thread.run(Thread.java:1447)
Caused by: java.lang.AssertionError: 57344 != 53248
	at org.h2.mvstore.RandomAccessStore.shrinkStoreIfPossible(RandomAccessStore.java:679)
	at org.h2.mvstore.RandomAccessStore.writeChunk(RandomAccessStore.java:377)
	at org.h2.mvstore.RandomAccessStore.writeChunk(RandomAccessStore.java:28)
	at org.h2.mvstore.FileStore.storeBuffer(FileStore.java:1524)
	... 6 more
2025-07-31 23:44:48.267041+02:00 jdbc[10]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
